.github/workflows/scheduled.yml

```diff
@@ -30,10 +30,11 @@ jobs:
         id: set-up-homebrew
         uses: Homebrew/actions/setup-homebrew@master
 
+      - name: Reset homebrew/cask tap
+        run: brew update-reset $(brew --repository homebrew/cask)
+
       - name: Update data for homebrew/cask
-        run: |
-          brew update-reset $(brew --repository homebrew/cask)
-          /usr/bin/rake cask
+        run: bundle exec rake casks
```

Comment:
Let's use `/usr/bin/rake` unless it's actually needed, it's a bit faster. If we can use it everywhere: all the better.

Of course, if it's not meaningfully faster and there's some other benefit in being consistent here: ignore me!

Reply:
Happy to use `/usr/bin/rake` everywhere unless it doesn't work. I just wanted the consistency and chose `bundle exec rake` over the others for no real reason. But yeah, if one doesn't work in a certain place I don't mind the inconcsistency

Reply:
Yeh, whichever works.